src/components/study-pane.js

```diff
@@ -777,6 +777,13 @@ const StudyPane = (props) => {
         }
     }
 
+    function scrollAndOpenVoltageLevel(vlId) {
```

Comment:
since showVoltageLevelDiagram, this breaks changing the url to reflect the currently shown sld